### PR TITLE
Refactor readme documentation

### PR DESCRIPTION
**Pull request by FalseR20**

### CODE REVIEW

**Review by AlexandrZagorskiy — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -16,340 +33,655 @@ $ pip install py-sc-client
...
+- *ScTemplateResult*.**size**()
+
+  The same, deprecated in version 0.3.0
+- *ScTemplateResult*[i: int]
+
+  Get ScAddr by index
+- *ScTemplateResult*.**get**(i: int)
+
+  The same, deprecated in version 0.3.0
+- *ScTemplateResult*.**get**(alias: str)
```
Comment (NikitaZotov) [inline README.md]: Don't deprecate get by index, it is needed when i want to get all addrs of elements in construction

File: README.md [hunk 2]
```diff
@@ -16,340 +33,655 @@ $ pip install py-sc-client
...
+```python
+from enum import Enum
+
+from sc_client.models import ScTemplateResult, ScAddr
+
+template_result: ScTemplateResult
+length = len(template_result)  # in the resulting construction
+template_result.size()  # deprecated count of elements, will be removed in version 0.3.0
+first_element = template_result[0]  # get an element from the result by index
+template_result.get(0)  # deprecated get by index, only str, will be removed in version 0.3.0
```
Comment (NikitaZotov) [inline README.md]: the same

File: README.md (hunk 1 above)
Comment (FalseR20) [reply README.md]: Removed deprecated mark

File: README.md (hunk 2 above)
Comment (FalseR20) [reply README.md]: Done

**Review by NikitaZotov — APPROVED** [review]

**Pull request merged**